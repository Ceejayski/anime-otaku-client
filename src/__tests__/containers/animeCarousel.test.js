import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import { create } from 'react-test-renderer';
import AnimeCarousel from '../../containers/animeCarousel';
import store from '../../redux';

test('renders Correctly', () => {
  const Placeholder = () => (
    <Provider store={store}>
      <BrowserRouter>
        <AnimeCarousel />
      </BrowserRouter>
    </Provider>
  );
  const tree = create(<Placeholder />).toJSON();
  expect(tree).toMatchSnapshot();
});
