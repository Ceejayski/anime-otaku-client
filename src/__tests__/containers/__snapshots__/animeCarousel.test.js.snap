// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Correctly 1`] = `
Array [
  <h3
    className="text-center"
  >
    Top Anime
  </h3>,
  <div
    className="w-100"
    id="animes"
  />,
]
`;
